docs/sources/features/datasources/cloudmonitoring.md

```diff
@@ -81,11 +81,11 @@ Click on the links above and click the `Enable` button:
 
 If Grafana is running on a Google Compute Engine (GCE) virtual machine, it is possible for Grafana to automatically retrieve default credentials from the metadata server. This has the advantage of not needing to generate a private key file for the service account and also not having to upload the file to Grafana. However for this to work, there are a few preconditions that need to be met.
 
-1. First of all, you need to create a Service Account that can be used by the GCE virtual machine. See detailed instructions on how to do that [here](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#createanewserviceaccount).
-1. Make sure the GCE virtual machine instance is being run as the service account that you just created. See instructions [here](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#using).
+1. First of all, you need to create a Service Account that can be used by the GCE virtual machine. For more information about create a Service Account, refer to [`create new service account`](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#createanewserviceaccount).
```

Comment:
```suggestion
1. First of all, you need to create a Service Account that can be used by the GCE virtual machine. For more information, refer to [Create new service account](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#createanewserviceaccount).
```

Reply:
When the "about ..." part is redundant with the title of the target, you can leave it out, as in this case.

Reply:
Hi @oddlittlebird, may I confirm "leave it out" means to follow the previous version use "about ..." redundant part with the title of the target?

Reply:
Correct

Reply:
okay thank you, sorry I delete previous message, just figured out how the suggestion works